package.json

```diff
@@ -171,10 +171,10 @@
         "title": "Run"
       },
       {
-        "command": "one.device.register",
-        "title": "Register",
+        "command": "one.device.refresh",
+        "title": "Refresh under this",
```

Comment:
`under this` doesn't sound technical wording.. As toolchainView's refresh button, how about `Refresh devices`?

Reply:
Oh, I think I remove that by mistake. I will fix it.